scripts/PearlPowderISIS/pearl_cycle_factory.py

```diff
@@ -0,0 +1,154 @@
+# pylint: disable=anomalous-backslash-in-string, too-many-branches, superfluous-parens
```

Comment:
The `anomalous-backslash-in-string` can actually be dangerous in paths and similar strings. We need to keep an eye on it and they should be removed in next PRs (after system tests are in place).